src/python/include/Alpha_complex_factory.h

```diff
@@ -103,38 +140,9 @@ class Inexact_Alphacomplex_dD final : public Abstract_alpha_complex {
 
  private:
   bool exact_version_;
```

Comment:
Does this make sense in the Inexact class?

Reply:
Indeed exact_version was useless in Inexact_alpha_complex_dD. Done on 72a56067